gateway/compose.production.yaml

```diff
@@ -300,5 +301,39 @@ services:
     #         - sds-gateway-prod-minio-net
     #         - sds-network-prod
 
+    jupyter:
+        # JupyterHub for interactive data analysis and notebooks (PRODUCTION)
+        build:
+            context: .
+            dockerfile: ./compose/production/jupyter/Dockerfile
+            args:
+                JUPYTERHUB_VERSION: latest
+        restart: always
+        image: sds-gateway-prod-jupyter
+        container_name: sds-gateway-prod-jupyter
+        networks:
+            - sds-network-prod
+        volumes:
+            - './compose/production/jupyter/jupyterhub_config.py:/srv/jupyterhub/jupyterhub_config.py:ro'
+            - '/var/run/docker.sock:/var/run/docker.sock:rw'
+            - 'sds-gateway-prod-jupyter-data:/data'
+        userns_mode: 'host'  # needs to match UID of the host machine in order to access the Docker socket
+        user: root  # docker socket requires root user
```

Comment:
we'll need to review this; if there's no way to run this without privileged mode, we might need to pick a different approach.

https://jupyter-docker-stacks.readthedocs.io/en/latest/using/recipes.html#using-sudo-within-a-container